Twilio/TwiML/TwiML.php

```diff
@@ -67,6 +67,19 @@ public function setAttribute($key, $value) {
         return $this;
     }
 
+	/**
+	 * @param string $name XML element name
+	 * @param string $value XML value
+	 * @param array $attributes XML attributes
+	 */
+	public function addChild($name, $value = null, $attributes) {
```

Comment:
I think if you have a default value for `$value` you need one for `$attributes` too (I would use `[]`, since we're locked to PHP 5.5+ now)